system-tests/projects/simple-ct/cypress.config.js

```diff
@@ -1,5 +1,7 @@
 module.exports = {
   component: {
+    experimentalSingleTabRunMode: true,
+    video: false,
```

Comment:
I needed to disable video for `simple-ct` so that the `run_ct_spec.js` system test spec passes for Webkit. If we try to take video on Webkit with single tab enabled, the browser just hangs once the specs are finished.

This is related to #23815 which we can look at later down the road of bringing `experimentalSingleTabRunMode` to GA.

Reply:
Can we add a comment to that extent here with the bug reference?

Reply:
added in https://github.com/cypress-io/cypress/pull/25257/commits/6bcc8da4310030c3bf031728bb7039188e0bd59b